libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -743,7 +743,7 @@ bool AP_BLHeli::BL_ConnectEx(void)
 
     blheli.interface_mode[blheli.chan] = 0;
 
-    uint16_t *devword = (uint16_t *)blheli.deviceInfo[blheli.chan];
+    uint16_t *devword = (uint16_t *)blheli.hw_deviceInfo[blheli.chan];
```

Comment:
doesn't look right, would need to be chan/2, but better to use functions that pull by byte

Reply:
done